src/containers/BookForm.js

```diff
@@ -10,18 +12,89 @@ const cat = [
   "Sci-Fi",
 ];
 
-const BookForm = () => (
-  <form>
-    <h3>Add a book</h3>
-    <br />
-    <input type="text" id="Title" placeholder="title" />
-    <br />
-    <select id="Category">
-      <option value={cat}>Category</option>
-      {cat.map((cat) => (<option key={cat.id}>{cat}</option>))}
-    </select>
-    <button type="submit">Confirm</button>
-  </form>
-);
-
-export default BookForm;
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: Math.round(Math.random() * 100),
+      title: "",
+      category: "",
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    event.preventDefault();
+    switch (event.target.id) {
+      case "Category":
+        this.setState({
+          category: event.target.value,
+        });
+        break;
+      case "Title":
+        this.setState({
+          title: event.target.value,
+        });
+        break;
+      default:
+        return this.state;
+    }
+    return false;
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    if (this.state.title === "" || this.state.category === "") {
+      alert("Please enter your book name and category"); // eslint-disable-line no-alert
```

Comment:
This should be production code, we shouldn't have `alert`, please remove the `alert`.